ui/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -262,12 +285,8 @@ export default function AwaitingSwap({
             );
           } else if (errorKey) {
             await dispatch(navigateBackToBuildQuote(history));
-          } else if (
-            isSwapsDefaultTokenSymbol(destinationTokenInfo?.symbol, chainId)
-          ) {
-            history.push(DEFAULT_ROUTE);
           } else {
-            history.push(`${ASSET_ROUTE}/${destinationTokenInfo?.address}`);
```

Comment:
Why is this line deleted?

Reply:
Great question! We synced up yesterday and here is a summary: the requirement is to show the `Close` button after a swap is done and that will redirect to the main page. However, before the swap is done, a user can click on `View in activity`: 
<img width="343" alt="image" src="https://user-images.githubusercontent.com/80175477/118964366-b7b82200-b967-11eb-890c-ffdc0ee70fb7.png">

Which redirects to the Activity tab, where a user can see progress of their transaction.

Reply:
An update.. I was redirecting to the Activity tab on the main page. I changed it back to the Activity tab on an asset's page, so it looks like this now: 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/80175477/119017009-d84d9f80-b99a-11eb-9cdb-4d32e979e994.png">